uppsrc/ide/SelectPkg.cpp

```diff
@@ -212,7 +213,6 @@ SelectPackageDlg::SelectPackageDlg(const char *title, bool selectvars_, bool mai
 	brief <<= THISBACK(SyncBrief);
 	search.NullText("Search (Ctrl+K)", StdFont().Italic(), SColorDisabled());
 	search << [=] { SyncList(Null); };
-	search.SetFilter(CharFilterDefaultToUpperAscii);
```

Comment:
I think this filter with making everything capital latter is fine and shouldn't be changed...